.changeset/nasty-pianos-lie.md

```diff
@@ -0,0 +1,5 @@
+---
+"@quri/ui": patch
+---
```

Comment:
Why did you make a patch for UI, but not for the Hub? Are you thinking changelogs more for the key dependencies? (lang, ui, components)?

Reply:
Hub doesn't get released on npm, so its `version` in `package.json` is meaningless.

In fact, `@quri/hub` is ignored in [config.json](https://github.com/quantified-uncertainty/squiggle/blob/main/.changeset/config.json), so `changeset` never offers to bump it.

Also, right now changeset _files_ are pretty useless, i.e. they don't automate anything useful.

There are two things they could affect:
1) versions (but those are pretty predictable, usually we touch on each package a bit in every release)
2) changelogs (but right now changelog generation is disabled in `config.json`, because we have a centralized `Changelog.mdx` in website that should be a source of truth)

I still use changeset files when I assemble release notes by hand.

I hope to revisit this setup after we settle on regular weekly/biweekly releases, and maybe re-enable changelogs generation, because it's still useful to attach a file to each NPM, or because it might be useful to generate a separate changelog for the Hub that's not included in squiggle-lang Changelog.mdx notes.

Reply:
Makes sense, thanks for clarifying!